File: usbTerminal/src/main/java/com/ismet/usbterminal/EmptyFragment.java
package com.ismet.usbterminal;

import android.graphics.Color;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class EmptyFragment extends Fragment{
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = new View(container.getContext());
        view.setBackgroundColor(Color.TRANSPARENT);
        return view;
    }
}

File: usbTerminal/src/main/java/com/ismet/usbterminal/EmptyFragment.kt
package com.ismet.usbterminal

import android.graphics.Color
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment

class EmptyFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ) = View(requireContext()).apply {
        setBackgroundColor(Color.TRANSPARENT)
    }
}